tests/rest/client/utils.py

```diff
@@ -147,12 +147,20 @@ def invite(self, room=None, src=None, targ=None, expect_code=200, tok=None):
             expect_code=expect_code,
         )
 
-    def join(self, room=None, user=None, expect_code=200, tok=None):
+    def join(
+        self,
+        room=None,
+        user=None,
+        expect_code=200,
+        tok=None,
```

Comment:
It would be nice to fill in these type hints while we are here! If that sets up crazy mypy work though then feel free to ignore this comment!